docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -1,19 +1,26 @@
 ## Developing a web viewer
 
-### Setup
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone imodeljs-samples repo](https://github.com/imodeljs/imodeljs-samples)
-- Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
+### Develop
+- [Install necessary prerequisites]($docs/getting-started/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`.

```

Comment:
Are these changes similar, or the same, as the one's @kckst8 made in the other PR?

Reply:
Whoops. I think I copied the same changes from that PR.